frontend/src/metabase/dashboard/components/add-card-sidebar/QuestionList.jsx

```diff
@@ -71,6 +92,15 @@ export function QuestionList({
                 )}
               />
             ))}
+            <PaginationControls
```

Comment:
Semantically it would be better to make like that:
```<>
    <SelectList>
    ...
    </SelectList>
    <PaginationControls />
</>``` 

that's because SelectList is `ul` and `QuestionListItem` is `li` and the pagniation element does not fit in there